control/pid.go

```diff
@@ -56,6 +56,7 @@ func (p *basicPID) Next(ctx context.Context, x []*Signal, dt time.Duration) ([]*
 			p.kI = p.tuner.kI
 			p.kP = p.tuner.kP
 			p.logger.CInfof(ctx, "Calculated gains are p: %1.6f, i: %1.6f, d: %1.6f", p.kP, p.kI, p.kD)
+			p.logger.CInfof(ctx, "You must MANUALLY ADD p, i and d gains to the robot config to use the values after tuning")
```

Comment:
should we add some slashes newlines and stars or something so its really obvious? also fine as is

Reply:
Added more yelling